.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+.test_md
```

Comment:
- [ ] We should update the `.test_md` in `.gitignore` for the actual name of the file that we want to exclude from repository, the name of the file is `test.md`.

Reply:
Thank you so much for this! I was trying to figure this out🙏🏾🙏🏾

Reply:
No matter what I punch into the gitignore file it won't remove itself from the directory

Reply:
- [ ] I can feel how you feel when you try to do something but it refuses, and I sincerely recognize your effort trying to make this `test.md` file disappeared from your repo. So a quick tip about this is:
  1. Firstly: It is refusing to be removed from there because its name in `.gitignore` file is not correctly spelled, it should be `test.md` instead of `~test.md/~` (note that it should not have the last slash => slashes are for directories not files)
  2. Secondly: After fixing its name from `.gitignore` run this command `git rm -f test.md` 
  3. Thirdly: Commit the changes and push, => from there the test.md file would have been removed completely from your repo, to have locally, just re-create it and it will no longer go on your repo